### ISSUE TITLE
EIP-2208: Thought Rewards Proposal for funding good thoughts & also leaders of good thoughts 

### ISSUE THREAD

**Issue by ryanseanadams** [ethereum/EIPs#2208]
---
eip: 2208
title: Thought Rewards Proposal for funding good thoughts & also leaders of good thoughts 
author: Ryan Sean Adams (@ryanseanadams)
discussions-to: https://github.com/ethereum/EIPs/issues/2208
status: Draft
type: Standards Track
category: Core
created: 2019-07-23
requires: NA
---

## Simple Summary

The efforts of thought leaders ("thot leaders") are essential to the protection and promotion of the value of ETH which is money used to pay for stuff in the Ethereum space. Currently these efforts receive zero ("0") compensation. 0 is too low of a number.

This proposal sets aside 0.03333 ETH per Ethereum block for 60 months (344,410 ETH) as a Thot Leader Block Reward ("TLBR") reserved for funding good thot leaders and good thots. 

Good thot leaders obviously include [@sassal0x](https://twitter.com/sassal0x), [@econoar](https://twitter.com/econoar), [@ameensol](https://twitter.com/ameensol), [@elonmusk](https://twitter.com/elonmusk), [@ryansadams](https://twitter.com/ryansadams) and others added to [the boat](https://twitter.com/LUKACACIC/status/1146207295489150976) through the impartial & public curation process of [@TheWolfofEthereum](https://twitter.com/LUKACACIC). Custody of funds will be stored in an impenetrable multisig only accessible by a set of trusted individuals who can be trusted. 

## Abstract

The technical issue this solves is that technically there is no reward for thot leadership. This EIP adds `0.03333 ETH` to the block reward for a specific distribution period of `10,333,333.3333333333 BLOCKS` (about 60 months). The `RECIPIENT_ADDRESS` is a dusty old ledger nano which will act as a hardware-secured landing zone for later transfer to the TLBR multisig wallet for the purpose of thot leader funding.  The emission schedule would start at the hard fork block number and continue for `10,333,333.3333333333 BLOCKS` (about 60 months) at which point the address and amount would return to 0. 

Each thot leader will deliver no less than 10 good thots per month. Thot leader tweets are public so there's 100% transparency & accountability.

Of course this is a one-time only block reward. Thot leaders will never require rewards again, that would be silly. If another set of thot leaders requests funds they should be harshly denied. Any further distribution would require a severe & protracted hard fork.

## Motivation

The motivation is that today thot leaders give a lot and they get 0 compensation. Think of your favorite memes. Chances are they were made by a thot leaders. DeFi? Thot leades. Money Legos? Thot leader. ETH is money? Also thot leaders. They did all this with 0 budget, imagine what they could do with more than 0.

With TLBR funding we'll have more good thots and more good leaders. It's possible (but not guaranteed) that these thots will result in a faster & more scalable ETH 2.0.

## Specification

Two constants will be introduced:

- `REWARD_DURATION_IN_BLOCKS`, which specifies the number of blocks after `ISTANBUL_BLOCK_HEIGHT` when the reward collection will terminate (i.e., at block `ISTANBUL_BLOCK_HEIGHT + REWARD_DURATION_IN_BLOCKS`)
- `BENEFICIARY_ADDRESSES`, a list of tuples containing the address and the amount to be transferred to this address per block. These amounts will be determined as the loan is collected from participating organizations and the addresses for repayment will be specified by them. The total of the amounts will sum to 0.0055. At the end of

```

REWARD_DURATION_IN_BLOCKS = 10,333,333.3333333333
TLBR_BLOCK_REWARD = 0.03333



BENEFICIARY_ADDRESSES = [
 (<address>, <amount_in_eth>),
 (<address>, <amount_in_eth>)
]
```

Beginning with block `ISTANBUL_BLOCK_HEIGHT`, the reward is added to the participating addresses within `BENEFICIARY_ADDRESSES` at each block until the end of the `REWARD_DURATION_IN_BLOCKS`.

```
IF (CURRENT_BLOCK - ISTANBUL_BLOCK_HEIGHT <= REWARD_DURATION_IN_BLOCKS)

FOR BENEFICIARY in BENEFICIARY_ADDRESSES:
  BENEFICIARY[0].balance += BENEFICIARY[1]

```

## Rewards Distribution Rationale
```
Total Block Reward Increase = `0.03333` seems like a fair number
```

*With a price of ETH at $211 this will raise USD $72,600,000 for thot leadership & good thots over the next 60 months! Wow!*

## Rationale

There's been lots of talk about block rewards for development.

Yeah, development is cool and stuff. But memes can 10x the value of a cryptocurrency a lot quicker. Thots produce memes, memes produce higher price, and higher price pays for development. So adding rewards for thots is actually the best way to pay for development.

**Budget**

Thot theme working groups:
 - ETH is money thots ($20m)
 - Money Legos & DeFi thots ($10m)
 - Open Finance thots ($5m)
 - Anti Maximalist FUD thots ($12m)
 - No, a Full Node Isn't 5 TBs thots ($25m) 

## Personal Notes and Disclosure 
I want to address concerns of conflicts of interest directly. I have none. As a thot leader, my interest is good thots. And I have zero conflicts on that interest.

### Disclaimer

I do not claim to represent the community with my views; however I do represent all good and clear-thinking people of the world. Therefore, any criticism of this proposal is likely false and unwarranted. 

#### 

## Copyright
Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).

**Comment by shemnon** [comment]
This is cute for a twitter troll, but for what should be a standards base repository this is wholly inappropriate.

**Comment by ryanseanadams** [comment]
> This is cute for a twitter troll, but for what should be a standards base repository this is wholly inappropriate.

Sorry you think that! Used 2025 as a template:
https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2025.md

**Comment by shemnon** [comment]
The difference is 2025 is done in good faith, and this is done in bad faith.  You don't really want this to pass.

**Comment by ryanseanadams** [comment]
> The difference is 2025 is done in good faith, and this is done in bad faith. You don't really want this to pass.

This was done in good faith.

It is intended to shed a light on the holes in the current process & the dangers of opening up politically contentious EIPs. When the limitations of EIPs become obvious to the community it will lead to higher quality EIPs & less spam.

**Comment by shemnon** [comment]
The intent is good faith, the manner is a net negative to the process.  You can make those points without polluting the EIP specifications with political banter.  There is a place for conversations like this.  A specification repository is not that place.  Because you posted a specification you don't actually want implemented is why I called it bad faith and an abuse of the process.

**Comment by shemnon** [comment]
If you have a political EIP that you might actually want passed, that is what should be posted.

**Comment by ryanseanadams** [comment]
> If you have a political EIP that you might actually want passed, that is what should be posted.

Like this one?
https://github.com/ethereum/EIPs/issues/960

**Comment by shemnon** [comment]
Actually, yes.

**Comment by ryanseanadams** [comment]
960 was submitted to spark conversation about fixed cap.

**Comment by ryanseanadams** [comment]
2025 was submitted to spark conversation about dev funding via block rewards (again!)

**Comment by shemnon** [comment]
The key difference was the tone.  It was pro of a particular position and not demeaning of another position.  You want to degrade, debase, and demean another contribution, use more appropriate venue like Twitter.  960 did no such degradation or disrespect of other contributions.  #960 has the decorum this proposal lacks.

**Comment by loredanacirstea** [comment]
@ryanseanadams , when do you plan on making a PR? I don't see a reason why this shouldn't be accepted as a Draft, under the current rules. Especially with the trend that editors shouldn't even bother making sure proposals are technically sound.

**Comment by pi0neerpat** [comment]
:+1: @loredanacirstea. Ok to merge (didn't check for markdown errors). Also lol!

**Comment by abrkn** [comment]
> What guys call girls in schools that send out nudes and porn of themselves. An acronym for That Ho Over There.

https://www.urbandictionary.com/define.php?term=THOT

**Comment by cyber-hokie** [comment]
Hi @ryanseanadams I see I am present in the boat. However I did not see a reference to William Shatner. Can you please expand on how the fund will be fairly distributed to include such individuals as celebrity endorsers? I once had a role in a Queen Latifah movie and in seasons 2 and 3 of Gossip Girl and would like to understand if I am eligible for such a category if it were to be added. Let’s speak offline in private, I have some ideas that might enhance the proposal, including the potential for a friendly fork with this already enabled rather than putting all our eggs into this basket, given the reception by many other stakeholders we don’t care about.

**Comment by GrzegorzWil** [comment]
Wow, I did something wrong. Again, the problems themselves. I do not know
anything about anything  fund and it probably is not about me .. I do not
know how much it is if I had to decide. part for new research development
projects. . to invest a bit. with a fruitful result. And certainly the
goals of charities . but fortunately I do not have such power and
capabilities.

sob., 27 lip 2019, 05:29 użytkownik Andrew <notifications@github.com>
napisał:
